docs/source/Installation.md

```diff
@@ -1,13 +1,13 @@
 # Installation
 
 ## Requirements
-The project leverages `docker-compose` with a custom python script so you need to have the following packages installed in your machine:
-* [docker](https://docs.docker.com/get-docker/) - v1.13.0+
-* [docker-compose](https://docs.docker.com/compose/install/) - v1.23.2+
+The project leverages `docker compose` with a custom Bash script and you need to have the following packages installed in your machine:
+* [docker](https://docs.docker.com/get-docker/) - v19.03.0+
+* [docker-compose](https://docs.docker.com/compose/install/) - v2.3.4+
 * [python](https://www.python.org/) - v3.6+
```

Comment:
shouldn't we able to remove this python dependency too? If I remember correctly, we needed it only to run the python start.py script

Reply:
> Python may still be required in order to run Django manage.py commands though.

But Django commands should be executed inside the `uwsgi` container where python is already installed there. So I think there won't be any real utility